imu:
  lp_alpha: 0.3
  threshold: 0.1

odom:
  inactive_timeout: 0.5
